<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="Led" name="led_v1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>V1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>35</width>
      <height>10</height>
     </rect>
    </property>
    <property name="text">
     <string>Valvula 1</string>
    </property>
   </widget>
   <widget class="cedaTank" name="tank_1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>83</x>
      <y>222</y>
      <width>35</width>
      <height>10</height>
     </rect>
    </property>
    <property name="text">
     <string>Tanque 1</string>
    </property>
   </widget>
   <widget class="Led" name="led_s12">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>150</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>S12</string>
    </property>
   </widget>
   <widget class="Led" name="led_s11">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>200</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>S11</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>204</y>
      <width>35</width>
      <height>10</height>
     </rect>
    </property>
    <property name="text">
     <string>S11</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>153</y>
      <width>35</width>
      <height>10</height>
     </rect>
    </property>
    <property name="text">
     <string>S12</string>
    </property>
   </widget>
   <widget class="cedaTank" name="tank_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>50</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Led" name="led_s22">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>50</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>S22</string>
    </property>
   </widget>
   <widget class="Led" name="led_s21">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>100</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>S21</string>
    </property>
   </widget>
   <widget class="QDial" name="bomba_principal">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>180</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>130</y>
      <width>35</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tanque 2</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>460</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Led</class>
   <extends>QToolButton</extends>
   <header>led.h</header>
  </customwidget>
  <customwidget>
   <class>cedaTank</class>
   <extends>QToolButton</extends>
   <header>cedatank.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
